server/opts.go

```diff
@@ -4036,6 +4036,7 @@ func ConfigureOptions(fs *flag.FlagSet, args []string, printVersion, printHelp,
 	fs.IntVar(&opts.Port, "p", 0, "Port to listen on.")
 	fs.StringVar(&opts.ServerName, "n", "", "Server name.")
 	fs.StringVar(&opts.ServerName, "name", "", "Server name.")
+	fs.StringVar(&opts.ServerName, "server_name", "", "Server name.")
```

Comment:
But just the line above there seem to be `-name` for the server name. Did I miss something?

Reply:
Just thought would be convenient since the error mentions `server_name` not `-name`.
```
jetstream cluster requires `server_name` to be set
```

Reply:
Right, sorry I did not pay enough attention before reviewing.